src/s3fs.cpp

```diff
@@ -5700,6 +5730,19 @@ int main(int argc, char* argv[])
         FdManager::InitFakeUsedDiskSize(fake_diskfree_size);
     }
 
+    // Set default value of free_space_ratio to 10%
```

Comment:
@ggtakec I chose this 10% value based on the strategy in juicefs: https://juicefs.com/docs/community/guide/cache/

The lowest possible value allows users to enjoy new features by default without preventing users from starting s3fs, because users generally reserve a certain amount of free space on the mount point.

Reply:
Is there any way to disable this ratio based space management? I'd like to set it to 0% as default, which means disabling this feature by default.

Reply:
@adamqqqplay Thank you for your discribe.
This part is only for messages at startup, so you can leave it at 10%.
In the future, if it becomes necessary to change ratio in practical use, I will make changes.

Reply:
@eryugey If you do not want to use the cache, s3fs will not use the cache unless you specify `use_cache`.
However, the disk is temporarily used when uploading.